ios/brave-ios/Sources/AIChat/Components/Extensions/AIChatView-Extensions.swift

```diff
@@ -0,0 +1,18 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import Foundation
+import SwiftUI
+
+extension View {
+  @ViewBuilder
+  public func hidden(isHidden: Bool) -> some View {
+    if isHidden {
+      self.hidden()
+    } else {
+      self
+    }
```

Comment:
This should be used with extreme prejudice and I'd recommend deleting it if possible. You're introducing view identify resets when the value you pass in changes. If you can get away with using the `opacity` modifier please do so